Bhakti Budiman Novanda

bhaktitud opened the pull request:
- basic route added
- still testing on routing

bhaktitud committed:
CLIENT ROUTE

- basic route added
- still testing on routing

bhaktitud committed:
dummy login to dashboard

bhaktitud committed:
checkpoint

bhaktitud committed:
login register connected to server but not using vuex yet

dhimashary commented on client/src/router/index.js:
```diff
@@ -0,0 +1,51 @@
+    name: 'Dashboard',
+    component: () => import('../views/Dashboard.vue'),
+    redirect: '/dashboard/products',
+    children: [
```
nanti yg create product masuk children sini juga ya

bhaktitud replied:
ok sudah mas

bhaktitud committed:
VUEX APPLIED

- Login Register migrated using basic vuex, still needs to be testing and refactor again

bhaktitud committed:
PRODUCT CREATE ROUTE

- sidebar fixed
- product container fixed
- create product added
- testing on vuex for the product route [on-progress]

bhaktitud committed:
add product feature

bhaktitud committed:
CRUD PRODUCT
Features :
- Fetch product
- Add Product
- Update product
- Delete product

bhaktitud committed:
logout functionality added

bhaktitud committed:
ui fixed

bhaktitud committed:
checkpoint

bhaktitud committed:
Update README.md

dhimashary commented on client/src/components/ErrorInfo.vue:
```diff
@@ -0,0 +1,15 @@
+<template>
+    <div class="alert alert-danger customSize fade show position-absolute" role="alert">
+        Oops.. there's something wrong with your input
```
bagusnya ya isi error infonya nge loop dari error message yang dikirim dari server

dhimashary commented on client/src/store/index.js:
```diff
@@ -0,0 +1,177 @@
+        state.loginStatus = true
+      } else {
+        state.loginStatus = false
+        localStorage.clear()
```
sebisa mungkin mutation bersih dari hal2 kayak gini ya, jadi isinya benar2 ngubah state doang

dhimashary commented:
client ketika berhasil register tidak diarahkan ke halaman login / langsung login

dhimashary commented:
delete tidak menggunakan konfirmasi
